migration/node-operators.md

```diff
@@ -0,0 +1,93 @@
+- Set the `network` to `main`
+- Choose your `sync_mode` setting as `full` or `history`
+- Enable the PoS RPC server by uncommenting the section in the `[rpc-server]` configuration
+
+**Step 2: Check Client Sync Status**
+
+**In the PoW chain**, ensure you are fully synced and in consensus. Start the PoW client with an RPC server (this might take a while). Run the following command:
+
+```bash
+node clients/nodejs/index.js --dumb --network=main --rpc=8648
+```
+
+**Step 3: Run the Activation Tool**
+
+The Activation Tool establishes a connection with the PoW chain via RPC, extracting data from your PoW client with the PoS client configuration.
+
+Ensure your PoW client is fully synced before running the Activation Tool on the PoS chain side. Before executing the Activation Tool, compile the **PoS chain** code by running `cargo build --release`.
+
+Once you are in consensus in the PoW chain, proceed to execute the Activation Tool by running the following command **in the PoS chain**, including the path to the configuration file containing your validator data and specifying the RPC server to be used:
+
+```bash
+cargo run --release --bin nimiq-pow-migration --url "http://127.0.0.1:8648" --config client.toml
+```
+
+:::
+
+::: details Users with an Existing PoW Client
+
+- Make sure you have your PoW RPC server enabled to allow your PoS client to connect to it
+- Clone the [PoS blockchain repository](https://github.com/nimiq/core-rs-albatross?tab=readme-ov-file#installation) and follow the instructions to compile the code
+
+**Step 1: Set Up your PoS Configuration File**
+
+This step involves configuring your PoS client settings to prepare for the transition. You'll set parameters such as the network, sync mode, and RPC server.
+
+Once you have your client compiled, you can:
+
+- Open the file `$HOME/.nimiq/client.toml` (see [example](https://github.com/nimiq/core-rs-albatross/blob/albatross/lib/src/config/config_file/client.example.toml))
+- Set the `network` to `main`
+- Choose your `sync_mode` setting as `full` or `history`
+- Enable the PoS RPC server by uncommenting the section in the `[rpc-server]` configuration
+
+**Step 2: Run the Activation Tool**
+
+The Activation Tool establishes a connection with the PoW chain via RPC, extracting data from your PoW client with the PoS client configuration.
+
+Ensure your PoW client is fully synced before running the Activation Tool on the PoS chain side. Before executing the Activation Tool, compile the **PoS chain** code by running `cargo build --release`.
+
+Once you are in consensus in the PoW chain, proceed to execute the Activation Tool by running the following command **in the PoS chain**, including the path to the configuration file containing your validator data and specifying the RPC server to be used:
+
+```bash
+cargo run --release --bin nimiq-pow-migration --url "http://127.0.0.1:8648" --config client.toml
```

Comment:
Same, this is assuming that the `PoS` client and server run in the same machine. And I guess some cases might be `https` instead.